inc/functions/get-categorized-blog.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
The current system could be called categorical to an extent. All template tags go in `template-tags.php`, for example. The main goal here is to keep files from becoming ridiculously long and to improve code organization. 

The "function per file" system also removes _some_ of the subjectivity from the code organization. If you need a new function, you create a new file instead of guessing which file it should go in (I know you still have to decide which directory to use, but work with me here 😄 ).

Reply:
I understand, but I am **strongly** against using a practice like this.

I do, however, think a `template-tags.php` file is the wrong way too, and instead it should be a directory, and you should use your best judgement to create logical grouped files of functions in there e.g. `template-tags/blog.php` or `template-tags/page-template-123.php` instead. The same can be said for `functions/` e.g. `functions/page-template-123.php` etc. 

Having to create a file for each function (or template tag) is just extra un-necessary steps that will become annoying to maintain and eventually ignored. I think exploring other logical groupings instead of this is a better option.